config/environments/test.rb

```diff
@@ -43,4 +43,7 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+  # config.middleware.use RackSessionAccess::Middleware
```

Comment:
this can go, we're no longer using RackSessionAccess so the config can't be just uncommented...